test/bdd/features/aries_router_e2e_sdk.feature

```diff
@@ -60,4 +60,58 @@ Feature: DIDComm Transport between two Edge Agents(without Inbound) through DIDC
     And   "Alice" waits for post state event "completed"
     And   "Bob" waits for post state event "completed"
     Then   "Alice" retrieves connection record and validates that connection state is "completed"
-    And   "Bob" retrieves connection record and validates that connection state is "completed"
\ No newline at end of file
+    And   "Bob" retrieves connection record and validates that connection state is "completed"
+
+  Scenario: Decentralized Identifier(DID) Exchange between two Edge Agents through Routers
```

Comment:
Should have a TODO where **both** WebSockets and HTTP are used.
(specifically showing the scenario where the agent communicates with their own router with WebSocket).

Reply:
[Added](https://github.com/hyperledger/aries-framework-go/compare/7aa1ef279c4908a1b7fc327eabe8f4fcb25bbc36..8b13eadcd144b44a492b85c3a400ab766a118cef#diff-6c628f852b19b92aa1ffe149b1485119R118) a TODO.